src/css/controls/imports/tooltip.less

```diff
@@ -76,6 +72,12 @@
 
     .jw-icon-volume .jw-overlay {
         visibility: visible;
+        outline: none;
+        transition: none;
```

Comment:
@waxidiotic why the `transition: none;` ? Do we see a flash without it ?

Reply:
yes